### ISSUE TITLE
www.openbioinformatics.org not working

### ISSUE THREAD

**Issue by dietmarfdz** [WGLab/doc-ANNOVAR#61]
I am trying to download a database and I get the following:

NOTICE: Web-based checking to see whether ANNOVAR new version is available ... Failed
NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg18_refGene.txt.gz ... Failed
NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg18_refGeneMrna.fa.gz ... Failed
NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg18_refGeneVersion.txt.gz ... Failed
WARNING: Some files cannot be downloaded, including http://www.openbioinformatics.org/annovar/download/hg18_refGeneVersion.txt.gz, http://www.openbioinformatics.org/annovar/download/hg18_refGene.txt.gz, http://www.openbioinformatics.org/annovar/download/hg18_refGeneMrna.fa.gz

If I check directly in the browser www.openbioinformatics.org it also does not work so I wonder the whole website is down for a reason....

Thanks,

**Comment by kaichop** [comment]
server is temporarily offline and we are addressing the issue.

**Comment by t-neumann** [comment]
Is there already some timeframe when the server will be back online?

**Comment by kaichop** [comment]
by end of Friday

**Comment by t-neumann** [comment]
I tried today and it is still down on my side.

**Comment by kaichop** [comment]
Not yet up, working on the issue

**Comment by t-neumann** [comment]
Can you foresee when the server will be online again? Just so I can plan accordingly

**Comment by t-neumann** [comment]
So the server seems to be up again to get to the download form, but I still cannot download any databases:

```
perl annotate_variation.pl -buildver hg38 -downdb -webfrom annovar exac03 hg38
NOTICE: Web-based checking to see whether ANNOVAR new version is available ... Done
----------------------------UPDATE AVAILABLE------------------------------
--------------------------------------------------------------------------
WARNING: A new version of ANNOVAR (dated 2018-04-16) is available!
         Download from http://www.openbioinformatics.org/annovar/
Changes made in the 2018-04-16 version:
         * added r.spl as a notation for certain indels based on HGVS guidelines
         * fixed a bug in calculating amino acid sequence when a block substitution covers both exon and intron
         * fixed a bug in classifying block substitutions as inframe mutations in coding_change.pl
         * coding_change.pl now handles gene names that contains dash or dot
         * fixed a bug table_annovar.pl when -dot2underline is used
         * shortened the protein sequence calculation for nonframeshift mutations in coding_change.pl
         * add -keepindelref argument in convert2annovar.pl to keep Ref/Alt allelles in VCF file
         * improved compatibility of coding_change.pl for certain gene names that contain @ and . characters
--------------------------------------------------------------------------
--------------------------------------------------------------------------
NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_exac03.txt.gz ... Failed
NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_exac03.txt.idx.gz ... Failed
WARNING: Some files cannot be downloaded, including http://www.openbioinformatics.org/annovar/download/hg38_exac03.txt.idx.gz, http://www.openbioinformatics.org/annovar/download/hg38_exac03.txt.gz
```

Is this a problem of me trying with an old version?

**Comment by lixin856** [comment]
I have submitted a form to request a download URL, but  I do not get the email within one hour(my email address is 18211510001@fudan.edu.cn).

**Comment by kaichop** [comment]
The DB are still being copied and most of them are not yet available (may
take a few weeks before this is completely done since many whole genome
files are >200G in size). But if you know a specific list of files (such as
hg38_exac03) that you need, I can copy it now.

**Comment by t-neumann** [comment]
@kaichop This would be very helpful actually. We are looking for `hg38` databases **only** and were thinking of using the following ones listed on the ANNOVAR website:

```
dbnsfp35c
dbscsnv11
intervar_20180118
exac03nontcga
gnomad211_genome
gnomad211_exome
1000g2015aug
avsnp150
clinvar_20190305
```

Is there a `hg38` version for `CADD` available? Then it would be great if you could provide also the latest version you have on that.

How does this work - would you let me know when they are copied and I can fetch them with the usual

```
perl annotate_variation.pl -buildver hg38 -downdb -webfrom annovar ...
```

command or you put them somewhere separate?

Thanks again for your help

**Comment by bharathramh** [comment]
@kaichop
I have tried downloading the databases today but still its down!!
Can I get hg19 databases of

> exac03
> gnomad211_exome
> gnomad211_genome
> clinvar_20190305

and can I expect in the upcoming update dbnsfp4.0 for hg19 and hg38??

**Comment by t-neumann** [comment]
And can you also provide the script

```
prepare_annovar_user.pl
```

somewhere?

**Comment by dietmarfdz** [comment]
Attached you can find the script. 
[prepare_annovar_user.zip](https://github.com/WGLab/doc-ANNOVAR/files/3152978/prepare_annovar_user.zip)

**Comment by t-neumann** [comment]
@dietmarfdz Thanks!

**Comment by kaichop** [comment]
@bharathramh all the four hg19 DBs listed below are available now.

**Comment by kaichop** [comment]
These DBs for hg38 are available now.

**Comment by bharathramh** [comment]
thankyou!! @kaichop

**Comment by t-neumann** [comment]
Thanks so much @kaichop !

**Comment by scottfrodsham** [comment]
@kaichop Thanks for getting the server back up and running. However, I'm having trouble downloading the  databases listed below for **hg38**. Any idea when these will be ready for download?

revel
knownGene

Thanks a bunch!

**Comment by kaichop** [comment]
@scottfrodsham These are ready now.

**Comment by scottfrodsham** [comment]
@kaichop Thank you!

**Comment by tanyaggarwal** [comment]
I am trying to download databases from annovar and I got the following:

NOTICE: Web-based checking to see whether ANNOVAR new version is available ... Failed
NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_refGene.txt.gz ... Failed
NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_refGeneMrna.fa.gz ... Failed
NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_refGeneVersion.txt.gz ... Failed
WARNING: Some files cannot be downloaded, including http://www.openbioinformatics.org/annovar/download/hg38_refGeneMrna.fa.gz, http://www.openbioinformatics.org/annovar/download/hg38_refGeneVersion.txt.gz, http://www.openbioinformatics.org/annovar/download/hg38_refGene.txt.gz

Can you please address what is the issue?

**Comment by kaichop** [comment]
It is working well but your server may have firewall issue. You can just
download it in your web browser instead then copy to your server.

**Comment by tanyaggarwal** [comment]
Ok. Thank you very much.

**Comment by tahirasma** [comment]
Greetings!

First of all kudos to all the amazing work.

I am encountering the same error (with refGenWithVer and clinvar_20210501): (Tried with multiple internet connections, the same error arise)

./annotate_variation.pl -downdb -buildver hg19 -webfrom annovar refGeneWithVer ./
NOTICE: Web-based checking to see whether ANNOVAR new version is available ... Failed
NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg19_refGene.txt.gz ... Failed
NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg19_refGeneMrna.fa.gz ... Failed
NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg19_refGeneVersion.txt.gz ... Failed
WARNING: Some files cannot be downloaded, including http://www.openbioinformatics.org/annovar/download/hg19_refGeneMrna.fa.gz, http://www.openbioinformatics.org/annovar/download/hg19_refGene.txt.gz, http://www.openbioinformatics.org/annovar/download/hg19_refGeneVersion.txt.gz

Looking forward for your support.

Regards,

**Comment by kaichop** [comment]
There is an issue with cloud hosting provider and the problem is now
solved. The website should work now.

**Comment by tahirasma** [comment]
Yes, it worked now. Thanks

**Comment by CV-engine-powerful** [comment]
I encountered a similar but not the same problem: The download process fails in about half of the attempts(3/5), but only **hg38_refGeneMrna.fa.gz** and not the other two.

../annotate_variation.pl -downdb -buildver hg38 -webfrom annovar refGene ./db

NOTICE: Web-based checking to see whether ANNOVAR new version is available ... Done
NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_refGene.txt.gz ... OK
NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_refGeneMrna.fa.gz ... Failed
NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_refGeneVersion.txt.gz ... OK
NOTICE: Uncompressing downloaded files
NOTICE: Finished downloading annotation files for hg38 build version, with files saved at the './db' directory
WARNING: Some files cannot be downloaded, including http://www.openbioinformatics.org/annovar/download/hg38_refGeneMrna.fa.gz

After several times of repeat I downloaded all files successfully. But I still would like to ask you what may the problem be?
